src/views/Correction.vue

```diff
@@ -1,120 +1,63 @@
 <template>
```

Comment:
I split up the previous Correction page into this, which does the preliminary fetch, and which displays one of the specific correction pages.

I'm not sure I love this but the alternative is separate, full correction pages, which means different routes, which I like even less.

Any better ideas? This is workable for now, should be extensible in the near future, and shouldn't be hard to refactor if needed.